### DISCUSSION TITLE
OSError: could not find or load spatialindex_c-64.dll

### DISCUSSION THREAD

**Discussion by AlexanderJuestel** [cgre-aachen/gemgis#171] [Installation] [answered]
This issue was originally asked and answered on StackOverflow: https://stackoverflow.com/questions/66372469/is-there-a-way-to-fix-this-import-error-for-geopandas

**Comment by RichardScottOZ** [comment]
Had that exact error recently.

**Reply by AlexanderJuestel** [reply]
It has never occurred to me for some reason 🤷‍♂️

**Comment by AlexanderJuestel** [comment, marked as answer]
Also here: https://github.com/geopandas/geopandas/issues/1812 

The solution is to set strict channel priorities to avoid the error:

conda config --env --add channels conda-forge
conda config --env --set channel_priority strict

As this is a proper solution, I will also add it to the documentation of GemGIS!
